src/Async.ts

```diff
@@ -5,3 +5,10 @@ type Run<A> = (abort: AbortSignal) => Promise<A>
 export class Async extends Effect('fx/Async')<Run<any>> { }
 
 export const run = <const A>(run: Run<A>) => new Async(run).returning<A>()
+
+export const never = run(() => new Promise<never>(() => { }))
+
+export const sleep = (ms: number) => run((abort) => new Promise<void>((resolve) => {
```

Comment:
As you could probably tell from the examples, I've implemented this about a dozen times, but have been hesitant to add it since it's bound directly to setTimeout.

That said, I don't really mind adding it now, and finding a way to deal with parameterized scheduling later.

Reply:
Yeah, some kind of Clock/Scheduler type seems somewhat inevitable once we focus more on testing scenarios